packages/@okta/vuepress-site/docs/guides/auth-js/main/index.md

```diff
@@ -0,0 +1,302 @@
+
+* Understand how to implement basic sign-in using the Okta Auth JavaScript SDK.
+* Understand basic installation and code configurations using the Okta Auth JavaScript SDK.
+* Implement the sample SPA use case and sign a user in to the application.
+
+**What you need**
+
+* [Okta Developer Edition organization](https://developer.okta.com/signup/)
+* Basic knowledge of building front-end JavaScript applications
+
+**Sample code**
+
+* [static-spa sample](https://github.com/okta/okta-auth-js/tree/master/samples/generated/static-spa)
+
+---
+
+## About the Okta Auth JavaScript SDK
+
+The Okta Auth JS SDK builds on top of the [Authentication API](/docs/reference/api/authn/) and [OpenID Connect API](/docs/reference/api/oidc/) to enable you to create a fully branded sign-in experience using JavaScript.
+
+The Okta Auth JS SDK is used by Okta's [Sign-in Widget](/code/javascript/okta_sign-in_widget/), which powers the default Okta sign-in page. If you're building a JavaScript front end or single-page app (SPA), the Auth JS SDK provides added control and customization beyond what is possible with the Widget.
+
+In this guide you'll learn how to use the Auth JS SDK with a simple static page application to authenticate and store an OpenID Connect (OIDC) token (`idToken`) and access token (`accessToken`).
+
+If you'd like to explore the entire Auth JS SDK, see the [Okta Auth JS JavaScript SDK](https://github.com/okta/okta-auth-js/#readme).
+
+## Installation
+
+Install the Auth JS SDK by linking out to the Okta CDN, or installing locally through `npm` or another package manager.
+
+### CDN
+
+To use the CDN, include this script in your target HTML page:
+
+```html
+<!-- Latest CDN production Auth JS SDK-->
+<script src="https://global.oktacdn.com/okta-auth-js/6.0.0/okta-auth-js.polyfill.js" type="text/javascript"></script>
+```
+
+More information is available in the [Okta Auth JS SDK](https://github.com/okta/okta-auth-js/#okta-auth-javascript-sdk).
+
+### npm
+
+```shell
+# Run this command in your project root folder.
+# yarn
+yarn add @okta/okta-auth-js
+
+# npm
+npm install @okta/okta-auth-js
+```
+
+More information is available in the [Okta Auth JS SDK](https://github.com/okta/okta-signin-widget#using-the-npm-module).
+
+## Add code to reference the SDK
+
+The following sections display basic code snippets you use when accessing the Auth JS SDK.
+
+### Initialize the SDK
+
+To initialize the SDK, create a new instance of the `OktaAuth` object. The `apps.js` file of the static-spa sample ues the function `createAuthClient()`:
+
+```JavaScript
+ function createAuthClient() {
+  // The `OktaAuth` constructor can throw if the config is malformed
+  try {
+    authClient = new OktaAuth({
+      issuer: config.issuer,
+      clientId: config.clientId,
+      redirectUri: config.redirectUri,
+      scopes: config.scopes,
+      tokenManager: {
+        storage: config.storage
+      },
+      transformAuthState,
+      recoveryToken: config.recoveryToken
```

Comment:
recoveryToken is an optional field and I don't think it needs to be shown for an authentication use case which we are describing here.

Reply:
Thanks, I'll remove.